charts/metal3/0.2.0/charts/ironic/templates/configmap.yaml

```diff
@@ -23,18 +23,17 @@ data:
   {{- end }}
   {{- if  ( .Values.global.enable_dnsmasq ) }}
   DNSMASQ_BOOT_SERVER_ADDRESS: {{ $ironicBootHost }}
-  DNSMASQ_DNS_SERVER_ADDRESS: {{ tpl .dnsmasqDnsServerAddress $ }}
+  DNSMASQ_DNS_SERVER_ADDRESS: {{ .Values.global.dnsmasqDNSServer }}
   DNSMASQ_DEFAULT_ROUTER: {{ .Values.global.dnsmasqDefaultRouter }}
   DHCP_RANGE: {{ .Values.global.dhcpRange }}
   {{- end }}
-  {{- with .Values.baremetaloperator }}
+  {{- if  .Values.debug.ironicRamdiskSshKey }}
+  IRONIC_RAMDISK_SSH_KEY: {{ .Values.debug.ironicRamdiskSshKey }}
+  {{- end }}
+  HTTP_PORT: "6180"
   CACHEURL: {{ $protocol }}://{{ $ironicCacheHost }}/images
   DEPLOY_KERNEL_URL: {{ $protocol }}://{{ $ironicBootHost }}/images/ironic-python-agent.kernel
   DEPLOY_RAMDISK_URL: {{ $protocol }}://{{ $ironicBootHost }}/images/ironic-python-agent.initramfs
-  {{- if  .dhcpHosts }}
-  DHCP_HOSTS: {{ .dhcpHosts }}
```

Comment:
Should we also remove the `baremetaloperator. dhcpHosts` from the [values.yaml](https://github.com/hardys/charts/blob/ironic_cleanups/charts/metal3/0.2.0/values.yaml#L122) file of the root chart?

Reply:
Oh yes, good catch - thanks!

Reply:
Fixed - I pushed as another commit as there were a number of other cleanups needed due to the recent changes.